app/main.py

```diff
@@ -32,13 +37,13 @@ def main() -> None:
     ]
```

Comment:
You have to change the wake_up_program and sleep_program blocks and their run — to the combination of the run_sequence and run_parallel. https://github.com/mate-academy/py-asyncio-iot#:~:text=You%20have%20to%20change%20the%20wake_up_program%20and%20sleep_program%20blocks%20and%20their%20run%20%E2%80%94%20to%20the%20combination%20of%20the%20run_sequence%20and%20run_parallel%20sending%20messages%20to%20the%20service%20to%20make%20it%20more%20logical.%20Please%20do%20that%20separately%2C%20firstly%20for%20wake_up%2C%20then%20for%20sleep.